MiniApp/Classes/core/Protocols/MiniAppMessageDelegate.swift

```diff
@@ -95,10 +106,23 @@ public class MAHostEnvironmentInfo: Codable {
     let platformVersion: String
     let hostVersion: String
     let sdkVersion: String
+    let hostLocale: String
 
-    public init(platformVersion: String, hostVersion: String, sdkVersion: String) {
+    public init?(platformVersion: String, hostVersion: String, sdkVersion: String, hostLocale: String) {
+        guard hostLocale.isValidLocale else { return nil }
```

Comment:
I don't think we need to return nil here. Is that a requirement?  We should still return the `sdkVersion`, `hostVersion` & `platformVersion` even if `hostLocale` is not found right?

Reply:
Also bit unsure here, I think there should not be a case where the hostLocale is nil because it will fallback to the default language if not set if we would not return nil we would have to think of how to handle the locale fallback for worst case. any suggestions?

Reply:
if `var getEnvironmentInfo: (() -> (MAHostEnvironmentInfo?))? { }` is overridden in Host app and  `miniapp.sdk.ios.locale` is not added in the Host app Localizable, then it will be nil.
If the method is not overridden then it won't be nil because SDK provides one.
Either you can remove this condition OR you can pass an empty string for `hostLocale` value OR send the default value from the SDK like below,

```
if !hostLocale.isValidLocale {
    self.hostLocale = MASDKLocale.localize(<LOCALE_KEY>)
}
```
What do you think ?

Reply:
added your suggestion 👍 👍